build.py

```diff
@@ -208,6 +208,7 @@ def do_make(target=None, log=False):
         print "Reading build variables from environment"
         publish = True
         use_environment = True
+        use_git = False
```

Comment:
Is it really an issue to have both turned on?

The way things work at the moment is that environment variables can be used to override values found first using git if both options are enabled.  At least it's used for bisection as each iteration lands on a different detached commit, so ``git_commit`` has to come from git while ``git_branch`` comes from the environment.

Reply:
Not quite sure.  I added that based on the request from @mattface.